udata/templates/dataset/display.html

```diff
@@ -387,7 +389,9 @@ <h3 class="text-left">{{ _('Producer') }}</h3>
         <div class="row">
             <div class="col-sm-9 list-group resources-list smaller">
             {% for resource in dataset.community_resources %}
-                {% include theme('dataset/resource/list-item.html') %}
+                <div itemprop="hasPart">
```

Comment:
example.com, lol

Yes I know it validate but as you see, you only have the name with part.
With distributions (which is exactly the resource concept), you have:
![screenshot-developers google com 2016-03-03 18-55-17](https://cloud.githubusercontent.com/assets/15725/13503803/a9e34e62-e171-11e5-89a6-94e8eb01ef05.png)